conda.recipe/pybert/meta.yaml

```diff
@@ -17,23 +17,24 @@ build:
 
 requirements:
   run:
-    - python >=3.8 <3.10
     - kiwisolver
-    - chaco =5.0.0.2
-    - pyibis-ami >=3.5.7
-    - scikit-rf
-    - pyside2
+    - numpy x.x
+    - pyibis-ami >=4.0.5
+    - pyside6
```

Comment:
Pyside2 -> Pyside6 was outside just numpy but works correctly on my end and seems like the right move overall.  https://docs.enthought.com/traitsui/changelog.html?highlight=pyside6#release-7-3-0